test/OpenTelemetry.Instrumentation.Dependencies.Tests/SqlClientTests.cs

```diff
@@ -34,8 +36,19 @@ namespace OpenTelemetry.Instrumentation.Dependencies.Tests
 {
     public class SqlClientTests : IDisposable
     {
+        /*
+            To run the integration tests, set the OT_SQLCONNECTIONSTRING machine-level environment variable to a valid Sql Server connection string.
```

Comment:
Lets use OTEL_SQLCONNECTIONSTRING, as OTel is the official acronym.
(https://github.com/open-telemetry/opentelemetry-specification/pull/700/files)

Reply:
Updated